packages/feedback/src/app/app.component.scss

```diff
@@ -0,0 +1,97 @@
+@import "../sass/component/_component.sass";
+@import "../sass/mixin/_mixin.sass";
+
+.app-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.page-container {
+  min-height: 600px;
+}
+
+.error-container {
```

Comment:
Why is this class defined here? It is not being used anywhere.

Reply:
Removed.